integration/query_array_compat_test.go

```diff
@@ -52,7 +52,7 @@ func TestQueryArrayCompatSize(t *testing.T) {
 			resultType: emptyResult,
 		},
 		"NotFound": {
-			filter:     bson.D{{"v", bson.D{{"$size", 4}}}},
+			filter:     bson.D{{"v", bson.D{{"$size", 5}}}},
```

Comment:
See in `composite.go`, I added 4th element to `ArrayDocuments` `array-three-documents`. So increasing it to 5.